packages/contracts-bedrock/src/L2/IOptimismSuperchainERC20.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+/// @title IOptimismSuperchainERC20
+/// @notice This interface is available on the OptimismSuperchainERC20 contract.
+///         We declare it as a separate interface so that it can be used in
+///         custom implementations of SuperchainERC20.
+interface IOptimismSuperchainERC20 {
```

Comment:
https://github.com/ethereum-optimism/optimism/pull/11042 also has an implementation of this interface, we will want to deduplicate them